packages/core/src/dialog/DialogHeader.css

```diff
@@ -1,22 +1,38 @@
 /* Styles applied to the root element */
 .saltDialogHeader {
-  padding-bottom: var(--salt-spacing-100);
+  padding-bottom: var(--salt-spacing-300);
```

Comment:
Changeset, this was highlighted by the community, that their existing header looks different than what's on the documentation, because our Site loads the lab CSS which overrides core